docs/user/README.md

```diff
@@ -20,6 +20,7 @@ For more details on these and other core concepts, you can learn about them [her
  - [Change Project Stack](changestack.md) - How to change a projects stack, for example to upgrade Node-RED.
  - [Logs](logs.md) - The Logs available in the FlowForge application.
  - [Project Link Nodes](projectnodes.md) - Custom nodes for sending messages between projects and devices.
+ - [Project Staging](project-staging.md) - Instruction on how to use FlowForge projects to build a DevOps staging pipeline.
```

Comment:
Let's talk about environments instead, staging is one environment. This seems very confusing to me

Reply:
Will change it to "Stages", rather than "Staging", as per:

- https://docs.aws.amazon.com/codepipeline/latest/userguide/concepts.html#concepts-stages
- https://learn.microsoft.com/en-us/azure/devops/pipelines/process/stages?view=azure-devops&tabs=yaml